set.mm

```diff
@@ -22712,11 +22729,11 @@ of the unique existential quantifier (note that ` y ` and ` z ` need not
        (New usage is discouraged.) $)
     2mo2OLD $p |- ( ( E* x E. y ph /\ E* y E. x ph ) <->
                           E. z E. w A. x A. y ( ph -> ( x = z /\ y = w ) ) ) $=
-      ( wex weq wi wal wmo eeanv jcab 2albii 19.26-2 19.23v albii alcom anbi12i
-      wa df-mo bitri 3bitri 2exbii 3bitr4ri ) ACFZBDGZHZBIZABFZCEGZHZCIZSZEFDFU
-      HDFZULEFZSAUFUJSHZCIBIZEFDFUEBJZUICJZSUHULDEKUQUMDEUQAUFHZAUJHZSZCIBIUTCI
-      ZBIZVACIBIZSUMUPVBBCAUFUJLMUTVABCNVDUHVEULVCUGBAUFCOPVEVABIZCIULVABCQVFUK
-      CAUJBOPUARUBUCURUNUSUOUEBDTUICETRUD $.
+      ( wex weq wi wal wa wmo exdistrv jcab 2albii 19.26-2 19.23v albii anbi12i
```

Comment:
The diff between 2mo2 and 2mo2OLD (created in #2972)  is that one uses exdistrv and the other eeanv.  So either delete 2mo2OLD, or keep it using eeanv

Reply:
Good catch. I removed it. Since there were many usages, I did a global "search and replace", so some "discouraged" proofs may have been affected, but the reason (if any) for their discouragement was not related to the use of eeanv/exdistrv.